### DISCUSSION TITLE
VITS d-vector issue

### DISCUSSION THREAD

**Discussion by Chaanks** [coqui-ai/TTS#2009] [General Q&A]
I'm training VITS on vctk dataset using d-vector but the quality is poor after 300k steps. I'm using default config for vctk recipe and YourTTS speaker representations. I have the same result with d-vector comming from an other speaker identification system.

`config.json`: 
[config.txt](https://github.com/coqui-ai/TTS/files/9632326/config.txt)

I'm attaching 2 samples from inference test on ADEPT dataset.

text: They fly to London!

https://user-images.githubusercontent.com/18086768/191928453-a8979f79-a47d-4ab2-a78c-97c568afd2e6.mp4

text: This train goes through Cambridge.

https://user-images.githubusercontent.com/18086768/191928461-d52367fc-87ee-4da0-b0f1-57a3c1ed238c.mp4

TensorBoard:

![attention](https://user-images.githubusercontent.com/18086768/191929702-851c6baf-4baf-48ec-add2-df80fc86c79a.png)

![loss](https://user-images.githubusercontent.com/18086768/191929731-31be65ff-5a31-40b1-a633-d0f3c931f1f2.png)

Thank you for any help you can offer !

**Comment by Ca-ressemble-a-du-fake** [comment]
In your config file you use phonemes with espeak but do not provide the phonemes list. Did you listen to the generated audios in TensorBoard ? Maybe you've encountered over-fitting, so can you try to listen to the evaluation audios at different steps in TensorBoard (AUdio tab) and check if the result was better at a previous step ?

**Comment by erogol** [comment]
I've not checked the config, but I think there is something wring with the character set.
